Gems/ScriptCanvas/Code/Include/ScriptCanvas/Grammar/AbstractCodeModel.cpp

```diff
@@ -3800,6 +3806,51 @@ namespace ScriptCanvas
             }
         }
 
+        bool AbstractCodeModel::HasUnparsedImplicitConnections(const Slot* outSlot, const Slot* inSlot)
+        {
+            if (inSlot->CreatesImplicitConnections())
+            {
+                // Get each endpoint connected to the input slot
+                EndpointsResolved connectedNodes;
+                if (const Node* inNode = inSlot->GetNode())
+                {
+                    connectedNodes = inNode->GetConnectedNodes(*inSlot);
+                }
+                for (const EndpointResolved& connectedNode : connectedNodes)
+                {
+                    // If the endpoint's slot is not the same as the output slot currently being processed, check if it has already been parsed
+                    if (connectedNode.second != outSlot)
+                    {
+                        // If there are no connections to ignore yet, that must mean there are still more slots on this node to parse
+                        if (m_parsedImplicitConnections.size() == 0)
+                        {
+                            m_parsedImplicitConnections.push_back(AZStd::make_pair(outSlot, inSlot));
+                            return true;
+                        }
+                        else
+                        {
+                            bool foundConnectionInParsed = false;
+                            for (auto connection : m_parsedImplicitConnections)
```

Comment:
Nit: Could use `const auto&` here so it doesn't make a local coppy
```suggestion
                            for (const auto& connection : m_parsedImplicitConnections)
```